tests/eas/load_tracking.py

```diff
@@ -216,3 +229,65 @@ def test_task_load_avg(self, experiment, tasks):
         this test does the same as test_task_util_avg but for load_avg.
         """
         return self._test_signal(experiment, tasks, 'load_avg')
+
+class CpuInvarianceTest(_LoadTrackingBase):
+    """
+    Goal
+    ====
+    Basic check for CPU invariant load tracking
```

Comment:
Hmm, I have always thought of "load tracking" as including both `load` and `util` (since they're both features of the "Per-entity load tracker"), but fair enough, I'll change it.